src/main/java/org/folio/innreach/domain/service/impl/LoanServiceImpl.java

```diff
@@ -46,6 +46,13 @@ public Optional<LoanDTO> find(UUID loanId) {
     return circulationClient.findLoan(loanId);
   }
 
+  @Override
+  public void delete(UUID loanId) {
+    circulationClient.findLoan(loanId)
+      .ifPresentOrElse(loanDTO -> circulationClient.deleteLoan(loanId)
+        ,()->log.info("Loan not found with loanId:{}",loanId));
```

Comment:
Same as above

Reply:
Think this should be reformatted to follow the change you made above to be:
`() -> log.info("Loan not found with loanId: {}", loanId));`

Notice how I'm adding spaces.

Reply:
Also move the comma to the previous line to be the same as line 45 above.